templates/Dockerfile.txt

```diff
@@ -1,11 +1,12 @@
-<% if(!coreclr){ %>FROM microsoft/aspnet:1.0.0-rc1-update1<% } %><% if(coreclr){ %>FROM microsoft/aspnet:1.0.0-rc1-update1-coreclr<% } %>
+<% if(!coreclr){ %>FROM microsoft/dotnet<% } %><% if(coreclr){ %>FROM microsoft/dotnet:core<% } %>
 
-RUN printf "deb http://ftp.us.debian.org/debian jessie main\n" >> /etc/apt/sources.list
-RUN apt-get -qq update && apt-get install -qqy sqlite3 libsqlite3-dev && rm -rf /var/lib/apt/lists/*
+#Set urls and port environments
+ENV ASPNETCORE_URLS="http://*:5000"
```

Comment:
Technically no templates are able to use ASPNETCORE_URLS AFAIK. I've hit that problem previously:
> The -p option is supported in dotnet-cli but --server-urls support needs to be determined
https://github.com/OmniSharp/generator-aspnet/pull/715
The solution is to modify web application startup either by:

- explicit use of `UseUrls`: https://github.com/aspnet/Templates/issues/574#issuecomment-220727859
- explicit use of 'hosting.json`: https://github.com/peterblazejewicz/recipes/blob/master/src/gulp-dotnet-browser-sync/StarterWeb/Program.cs#L12-L13

The first blog post about RC2 + Docker also has to use that implementation: http://laurentkempe.com/2016/05/16/ASP-NET-Core-RC2-Docker-and-HipChat-Connect-add-on/